test/integration/listener_lds_integration_test.cc

```diff
@@ -738,6 +738,52 @@ class ListenerFilterIntegrationTest : public testing::TestWithParam<Network::Add
           stat_prefix: tcp_stats
           cluster: cluster_0
 )EOF") {}
+
+  void createLdsStream() {
+    AssertionResult result =
+        fake_upstreams_[1]->waitForHttpConnection(*dispatcher_, lds_connection_);
+    RELEASE_ASSERT(result, result.message());
```

Comment:
Now that you are here, can you switch to ASSERT_TRUE(result)? ASSERT_TRUE/EXPECT_TRUE is more friendly in test case.

Reply:
got it